research/picai/nnunet_scripts/transfer_train.py

```diff
@@ -0,0 +1,277 @@
+"""Automates simple transfer learning for nnUNetv2
+
+    Assumptions:
+        - nnUNetv2 must be properly set up (ie. required environment variables set)
+        - The finetuning dataset must be properly formatted and in the nnUNet_raw folder
+        - A plans file for the pretrained model must be present in the nnUNnet_preprocessed
+            under the folder of the dataset it was pretrained on.
+        - The dimensions and voxel spacings of the finetuning dataset must
+            be the same as the pretraining dataset
+        - The pretrained model must already be saved and stored as a pytorch ckpt
+"""
+
+import argparse
+import os
+import shutil
+from multiprocessing import Process
+
+import torch
+from batchgenerators.utilities.file_and_folder_operations import load_json, save_json
+from nnunetv2.experiment_planning.plan_and_preprocess_api import extract_fingerprints, preprocess_dataset
+from nnunetv2.experiment_planning.plans_for_pretraining.move_plans_between_datasets import move_plans_between_datasets
+from nnunetv2.paths import nnUNet_preprocessed, nnUNet_raw
+from nnunetv2.run.run_training import run_training
+from nnunetv2.utilities.dataset_name_id_conversion import convert_id_to_dataset_name
+
+
+def setup_device(device: str) -> torch.device:
+    """Checks the device being used, sets the number of threads and returns a pytorch device
```

Comment:
I don't even think we need to have some documentation for some stuff in the research folder if it speaks for itself (not trying to make extra work), but just for future reference, we have slightly different documentation practices. Here is where you can get more information https://github.com/VectorInstitute/FL4Health/blob/main/CONTRIBUTING.MD